plugin/adsk/plugin/ProxyShape.cpp

```diff
@@ -69,4 +74,18 @@ void ProxyShape::postConstructor()
 #endif
 }
 
+#if defined(WANT_UFE_BUILD)
+SdfLayerRefPtr ProxyShape::computeRootLayer(MDataBlock& dataBlock, const std::string& filePath)
+{
+    auto rootLayerName = dataBlock.inputValue(rootLayerNameAttr).asString();
+    return LayerManager::findLayer(UsdMayaUtil::convert(rootLayerName));
+}
+
+SdfLayerRefPtr ProxyShape::computeSessionLayer(MDataBlock& dataBlock)
+{
+    auto sessionLayerName = dataBlock.inputValue(sessionLayerNameAttr).asString();
+    return LayerManager::findLayer(UsdMayaUtil::convert(sessionLayerName));
+}
+#endif
+
```

Comment:
As in the previous PR, this is how a supported proxy shape can respond to the compute in the base proxy with the serialized layers.